src/components/sound.js

```diff
@@ -24,44 +23,33 @@ module.exports.Component = registerComponent('sound', {
 
   update: function (oldData) {
     var data = this.data;
-    var diffData = diff(oldData || {}, data);
     var el = this.el;
     var sound = this.sound;
-    var src = data.src;
-    var srcChanged = 'src' in diffData;
+    var srcChanged;
+
+    oldData = oldData || {};
```

Comment:
I don't think we do. That might have been the original intent. But now it's better to use a flag for that.